docs/roman/linearity/description.rst

```diff
@@ -0,0 +1,56 @@
+Description
+============
+
+Assumptions
+-----------
+(When the superbias step is implemented in the pipeline, text about and
+a link to it will be added here).
```

Comment:
I'd remove the reference to superbias. Once INS has a linearity correction that depends on the bias subtraction we can add it back.